### ISSUE TITLE
"Create Notebook" button after /newNotebook does not work

### ISSUE THREAD

**Issue by markodraisma** [microsoft/vscode-copilot-release#1806]
Type: <b>Bug</b>

In Github Copilot Chat, ask @workspace /newNotebook predict iris species using online sklearn dataset and seaborn

The response is comming, but the Create Notebook button is not doing anyting

Extension version: 0.21.1
VS Code version: Code 1.94.1 (e10f2369d0d9614a452462f2e01cdc4aa9486296, 2024-10-05T05:44:32.189Z)
OS version: Windows_NT x64 10.0.22631
Modes:
Remote OS version: Linux x64 5.15.153.1-microsoft-standard-WSL2

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|12th Gen Intel(R) Core(TM) i7-1255U (12 x 2611)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled<br>webnn: disabled_off|
|Load (avg)|undefined|
|Memory (System)|15.64GB (4.48GB free)|
|Process Argv|--remote=wsl+Debian --crash-reporter-id d48b4e35-b0e1-4008-8403-4807f860b107|
|Screen Reader|no|
|VM|40%|

|Item|Value|
|---|---|
|Remote|WSL: Debian|
|OS|Linux x64 5.15.153.1-microsoft-standard-WSL2|
|CPUs|12th Gen Intel(R) Core(TM) i7-1255U (12 x 0)|
|Memory (System)|7.58GB (5.92GB free)|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vspor879:30202332
vspor708:30202333
vspor363:30204092
vscod805:30301674
binariesv615:30325510
vsaa593:30376534
py29gd2263:31024239
c4g48928:30535728
azure-dev_surveyone:30548225
a9j8j154:30646983
962ge761:30959799
pythongtdpath:30769146
pythonnoceb:30805159
asynctok:30898717
pythonmypyd1:30879173
2e7ec940:31000449
pythontbext0:30879054
accentitlementsc:30995553
dsvsc016:30899300
dsvsc017:30899301
dsvsc018:30899302
cppperfnew:31000557
dsvsc020:30976470
pythonait:31006305
dsvsc021:30996838
01bff139:31013167
a69g1124:31058053
dvdeprecation:31068756
dwnewjupytercf:31046870
2f103344:31071589
impr_priority:31102340
nativerepl2:31139839
refactort:31108082
pythonrstrctxt:31112756
wkspc-onlycs-t:31132770
wkspc-ranged-t:31151552
cf971741:31144450
defaultse:31146405
iacca2:31150323
notype1:31151523
5fd0e150:31155592

```

</details>

**Comment by markodraisma** [comment]
It used to work in previous versions of VSCode, Chat and the Jupyter Notebook extension.

**Comment by shubhnitb95** [comment]
@markodraisma - Did you get any resolution for this?

**Comment by markodraisma** [comment]
@shubhnitb95 I have not yet received or found any resolution for this.

**Comment by shubhnitb95** [comment]
@markodraisma - Yeah, it's still not working, for me as well.

**Comment by kbdharun** [comment]
Hi, I can confirm this issue is still present in copilot-chat 0.21.2 and is reproducible across systems.

**My system and extension information:**

![Image](https://github.com/user-attachments/assets/7f8d943e-2a34-47ba-9aa9-1096c11e0a6b)

![Image](https://github.com/user-attachments/assets/bbc864b6-f6ea-46f7-8859-906528a0a854)

**Comment by markodraisma** [comment]
In the new 1.95 version of VS Code this issue seems to be resolved. I'm now able to use the `/newNotebook` command again.
@kbdharun @shubhnitb95 @mjbvz Is this true for you too? Than this issue can be closed.

**Comment by kbdharun** [comment]
> In the new 1.95 version of VS Code this issue seems to be resolved. I'm now able to use the `/newNotebook` command again. [@kbdharun](https://github.com/kbdharun) [@shubhnitb95](https://github.com/shubhnitb95) [@mjbvz](https://github.com/mjbvz) Is this true for you too? Than this issue can be closed.

Yep, can confirm the option works now after updating VSCode and the extension to the latest version.
